src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -61,12 +63,12 @@ export default function SegmentedReqProvider(es, Private, Promise, timefilter, c
       // parameters via the handle
       if (_.isFunction(this._initFn)) this._initFn(this._handle);
       return this._createQueue().then((queue) => {
+        if (this.stopped) return;
+
         this._all = queue.slice(0);
 
         // Send the initial fetch status
         this._reportStatus();
-
-        return super.start();
```

Comment:
Thankfully the [`AbstractRequest#start()`](http://github.com/spalger/kibana/blob/8fef03b816e016a1637a2557c1a1cb6e0cab7bc2/src/ui/public/courier/fetch/request/request.js#L26-L42) method is pretty simple and really only exists for reporting purposes, to set this flag, and to make sure that requests aren't being executed multiple times.